src/components/CameraPermissions/__tests__/withPermissionsFlow.test.tsx

```diff
@@ -47,7 +47,7 @@ describe('CameraPermissions', () => {
         const wrapper = mount(
           <MockedReduxProvider>
             <MockedLocalised>
-              <WrappedComponent {...defaultProps} />
+              <WrappedComponent {...defaultProps} buttonType="none" />
```

Comment:
Why is the button type "none"? Where are you handling this "none" buttonType?

Reply:
Oh interesting. I don't think I made it on purpose. Might be there accidentally during merges. Reverted

Reply:
Fair enough, but why not removing `buttonType="none"` entirely and allow for nullish value?

Reply:
I see, that makes sense